generator/uscore/uscore_generator.rb

```diff
@@ -158,8 +158,15 @@ def create_search_test(sequence, search_param)
 
               skip 'No resources appear to be available for this patient. Please use patients with more information.' unless @resources_found
 
-              @#{sequence[:resource].downcase} = reply.try(:resource).try(:entry).try(:first).try(:resource)
+              @#{sequence[:resource].downcase} = reply&.resource&.entry&.first&.resource
               @#{sequence[:resource].downcase}_ary = reply&.resource&.entry&.map { |entry| entry&.resource }
+              page_count = 1
+              next_bundle = reply&.resource&.next_bundle
+              until next_bundle.nil? || page_count == 100
```

Comment:
I think 100 is pretty high. I feel like 500ms is a fairly typical response time for a FHIR search, which would mean potentially 50 seconds per sequence just on searching.

Reply:
Good point. I changed it to 20 since that seems like a more reasonable amount.